Upgrading.md

```diff
@@ -33,7 +33,9 @@ Install Python3 and from the folder where the binary resides (make sure `tasmota
 ```
 python -m http.server 8000
 ```
-Change your OtaUrl to http://ipoftheserver:8000/yourbinary.bin and start the upgrade process.
+Change your OtaUrl to http://ipoftheserver:8000/yourbinary.bin and start the upgrade process. Note: do not use `/`, `-`, or `.` characters in the name of `yourbinary`.
+
+If your binary build (yourbinary.bin) is larger than the available free flash program space, Tasmota will need to first install the minimal version of Tasmota to make more space. To have this work via the web server OTA process, you have to copy the file `tasmota-minimal.bin` in the same folder where `OTAURL` for `yourbinary.bin` is placed.
```

Comment:
Dear @meingraham, thanks for approving my PR. I'm not sure .... but I think the minimal binary file in OTAURL, should be name yourbinary-minimal.bin   ... to be tested if the name is tasmota-minimal.bin can work .... I will try to do the test in submit another PR if needed ! Regards.

Reply:
yes you are correct it has to be named yourbinary-minimal.bin and cannot have special characters. this was tested some time ago by me